docusaurus/docs/operate/cheat_sheets/full_node_cheatsheet.md

```diff
@@ -23,9 +25,20 @@ See the [Full Node Walkthrough](../walkthroughs/full_node_walkthrough.md) if you
 
 ## Pre-Requisites & Requirements
 
-1. **Linux-based System**: Ensure you have a Debian-based Linux distribution.
-2. **Root or Sudo Access**: You need administrative privileges to run the installation script.
-3. **Dedicated Server or Virtual Machine**: Any provider should work (e.g. Vultr).
+1. **Linux-based System**: Preferably Debian-based distributions (Ubuntu, Debian).
+2. **Hardware Requirements**:
+   - 4+ CPU cores
+   - 8+ GB RAM
+   - 200+ GB SSD storage (for chain data)
```

Comment:
Just FYI - this is something we're going to change soon - the numbers will be different depending on the network.

Reply:
I'm thinking we could also publish this data when taking snapshots and we'll have to think of another interview question..

Reply:
Trick question: our team is good enough that with AI, we'll never need to hire anyone ;)